src/RequestProcessor/Dmr/DmrService.cs

```diff
@@ -77,10 +80,14 @@ private static HttpRequestMessage CreateRequestMessage(DmrRequest request)
             };
 
             requestMessage.Headers.Add(Constants.XMessageIdHeaderName, request.Headers.XMessageId);
+            requestMessage.Headers.Add(Constants.XMessageIdRefHeaderName, request.Headers.XMessageIdRef);
             requestMessage.Headers.Add(Constants.XSendToHeaderName, request.Headers.XSendTo);
             requestMessage.Headers.Add(Constants.XSentByHeaderName, request.Headers.XSentBy);
             requestMessage.Headers.Add(Constants.XModelTypeHeaderName, request.Headers.XModelType);
 
+            // Unless specified by the caller - the use the text/plain mime type.
```

Comment:
Should this be `then use the text/plain mime type`?

Reply:
Yes - I forgot how to English.